runtime/near-test-contracts/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.0.0"
 publish = false
 authors = ["Near Inc <hello@nearprotocol.com>"]
 # Please update rust-toolchain.toml as well when changing version here:
-rust-version = "1.64.0"
+rust-version = "1.65.0"
```

Comment:
drive by, but specifying rust-version for test contracts doesnt' really makes sense, these crates shond basically never be published to crates.io

Reply:
would you suggest to just leave it unspecified? i.e. just delete this line?

Reply:
Yeah, that'd be better, but we also need to make sure that `themis` doesn't complain.